### PR TITLE
Added floating point vector addition test for cuda regression

### PR DESCRIPTION
**Pull request by bornaehsani**
Passes F1.

**Should be merged together with PR # 90 in bsg_manycore.**

### CODE REVIEW

File: cl_manycore/regression/cuda/test_float_vec_add.c [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+void host_float_vec_add (float *A, float *B, float *C, int N) { 
+	for (int i = 0; i < N; i ++) { 
+		C[i] = A[i] + B[i];
+	}
+	return;
+}
+
+
+int kernel_float_vec_add () {
+	bsg_pr_test_info("Running the CUDA Floating Point Vector Addition Kernel on a 1x1 grid of 2x2 tile group.\n\n");
```
Comment (drichmond) [inline cl_manycore/regression/cuda/test_float_vec_add.c]: Can you line split down to 80 characters?

Comment (drichmond) [reply cl_manycore/regression/cuda/test_float_vec_add.c]: Here's a good reason why. I shouldn't need to scroll sideways to read your code...

image

Comment (bornaehsani) [reply cl_manycore/regression/cuda/test_float_vec_add.c]: Very well, I've modified the test file to limit to 80 characters. Let me know if this looks good, and I'll apply the same to all other tests. Thanks Dustin!

**Comment by drichmond** [comment]
Perfect! 

This is fine for now - no need to redo all of them. We'll fix them when we change them for the no-dram mode (which may become an argument, not a macro)

**Comment by bornaehsani** [comment]
Great! OK will do.

**Pull request merged**